cmd/state-installer/test/integration/installer_int_test.go

```diff
@@ -31,13 +31,12 @@ func (suite *InstallerIntegrationTestSuite) TestInstallFromLocalSource() {
 	// Run installer with source-path flag (ie. install from this local path)
 	cp := ts.SpawnCmdWithOpts(
 		ts.InstallerExe,
-		e2e.WithArgs(target, "--source-path", ts.Dirs.Bin),
+		e2e.WithArgs(target, "--source-path", ts.Dirs.Bin, "--from-deferred"),
```

Comment:
Note this is again working around the actual failure. It doesn't look like this test is broken, it looks like what it is testing is.

The target location should be empty. It seems for some reason the installer is detecting that the target is non-empty and that the State Tool is already installed there, and giving the update flow as a result.

Reply:
If you know what to do, feel free to grab this PR or open a separate one. I'm juggling between a lot of tasks right now and am spinning my wheels on this one.

Reply:
Ok, just unstart it on Jira so someone else can take it.